src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -51,6 +52,43 @@ public class SpatialCoordinateSystemManager : Singleton<SpatialCoordinateSystemM
         private HashSet<INetworkManager> networkManagers = new HashSet<INetworkManager>();
         private ISpatialLocalizationSession currentLocalizationSession = null;
 
+        /// <summary>
+        /// Current Tracking state for the AR/VR Device associated with the application.
+        /// </summary>
+        public TrackingState TrackingState
+        {
+            get
+            {
+                if (!lookedForTrackingObservers)

```

Comment:
Can the Tracking observers register with the manager instead?

Reply:
yeah that sounds better than a single lookup